code-challenges.test.js

```diff
@@ -13,9 +16,24 @@
 
 // Reminder: The test will call your function
 
+// 
+
 
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
+//pseudo
+// make my function going with shuffleMe
+// probably going to use math.random somehow and iteration  
+// use pop to remove the first index and store it in a new var
+// use map to randomize new var
+//scratch that map doesnt like randomizing things looking up fisher yates for loop method
+// got scared only saw a return of 1 item in the array chainf my method from shift to slice
+// sweet gangs all here 
+// now the fun part lets make a test.. thinking im going to use .include perhaps
+// going to make two test one that ensures all four colors by using sort first to de randomize
+// not sure how to test if it has been randomized but i can compare it to its starting point to make sure at the very least the first index has been removed
+// feel very meh about this test what wouldve been a better way?
```

Comment:
There's a few ways to go about this. The simplest way is to create circumstances that you can replicate. If I get a random array of words and run sort on it I should get the same order of words if I sort them from a hard coded input. 
```expect(function(argument).sort).toEqual(array.sort())```